src/main/java/org/apache/maven/resolver/internal/ant/AntRepoSys.java

```diff
@@ -649,11 +649,10 @@ private Properties getEnvProperties( Properties props )
         {
             props = new Properties();
         }
-        boolean envCaseInsensitive = OS_WINDOWS;
         for ( Map.Entry<String, String> entry : System.getenv().entrySet() )
         {
             String key = entry.getKey();
-            if ( envCaseInsensitive )
+            if ( OS_WINDOWS )
```

Comment:
But previously the code would suggest we are interested in environment being or not case insensitive. But now it's only about being Windows. I think the readability is lowered now.

Reply:
> But previously the code would suggest we are interested in environment being or not case insensitive. But now it's only about being Windows. I think the readability is lowered now.

Created a variable and don't use it make the code more complex and harder to follow.   If the idea is modify the variable in the middle, should be do, but in the midtime, cause noise instead of help. of course that IMO.

Reply:
What if there was new final static like `CASE_INSENSITIVE_ENV = OS_WINDOWS` introduced, that would serve the old purpose of giving the reason for changes to `key` and would also allow to remove this local?

Reply:
> What if there was new final static like `CASE_INSENSITIVE_ENV = OS_WINDOWS` introduced, that would serve the old purpose of giving the reason for changes to `key` and would also allow to remove this local?

Don't you think would be better a comment in the code explaining the use case? IMO create another var it's the same because always will have the same result as `Os.isFamily( "windows" );`

Reply:
> Don't you think would be better a comment in the code explaining the use case?

No, I don't.

Reply:
Add a new variable it's the same. Redundant.